samples/nextjs/scripts/generate-config.ts

```diff
@@ -49,7 +47,8 @@ const config = {};\n`;
   });
   // Set computed values
   Object.keys(computedConfig).forEach((prop) => {
-    configText += `config.${prop} = ${computedConfig[prop]};\n`;
+    // eslint-disable-next-line prettier/prettier
+    configText += `config.${prop} = process.env.${constantCase(prop)} || ${computedConfig[prop]};\n`;
```

Comment:
The code that generates temp/config is already checking for environment variables; does this check need to be repeated here?

Reply:
@anastasiya29 Not sure I follow. This is the code that generates the temp/config. Previously, we only generated the environment variable portion of the line (`process.env.${constantCase(prop)} || `) for _non-computed_ properties. Here, we're adding that capability for computed properties as well (at the moment, that's just our `graphQLEndpoint`).